testsuite/integration-arquillian/servers/auth-server/jboss/common/vault/master_smtp__key

```diff
@@ -0,0 +1 @@
+secure_master_smtp_secret
```

Comment:
I don't like these files are doubled. Can we add those into standalone/configuration in testing phase (somewhere in base/pom.xml) or can we use files from server/auth-server/... in keycloak-server.json for undertow?

Reply:
Good point, I will see if I can update the poms to avoid duplication.

Reply:
This would be nice to have but not strictly necessary.